### PR TITLE
Improve air-gapped containers visibility in docs

### PR DESCRIPTION
**Pull request by ctalledo**
## Description

Improved the visibility of the air-gapped containers feature in the docs by:

* Creating a dedicated section for it under Hardened Desktop (before it was hiding inside Hardened-Desktop->Settings-Management).
* Adding it to the grid in the Hardened Desktop section.
* Adding it to the grid in the Security section.

NOTE: need help to create a proper icon for the air-gapped containers feature in the Hardened Desktop grid.

## Reviews

- [ ] Technical review
- [X] Editorial review
- [ ] Product review

### CODE REVIEW

**Review by djs55 — APPROVED** [review]
Thanks, the preview looked good to me!

**Comment by ctalledo** [comment]
> Also curious about where the need for this PR originated from? Were users complaining it wasn't visible enough?

Hi @aevesdocker, this originated internally, from our desire to market Air-Gapped Containers as a key component of Hardened Desktop. The docs structure must reflect this, and it wasn't prior to this change.

File: content/desktop/hardened-desktop/_index.md [hunk 1]
```diff
@@ -45,17 +49,20 @@ It is for security conscious organizations who:
 ### What does Hardened Docker Desktop include?
 
 It includes:
+
+{{< grid >}}
```
[the remark this answers is not in the document]
Comment (ctalledo) [reply content/desktop/hardened-desktop/_index.md:53]: Sure can do, but let me say: as a reader, I found it ** much clearer ** to read the grid compared to bulleted text. It immediately tells me what are the components of the feature, and I can then click on each. 

Thus I would encourage moving the grids up higher in the other pages to increase their visibility and make it easier for readers, rather than leaving them at the bottom where they are less helpful / visible.

File: content/desktop/hardened-desktop/_index.md [hunk 2]
```diff
@@ -45,17 +49,20 @@ It is for security conscious organizations who:
...
+
 - Settings Management, which helps admins to confidently manage and control the usage of Docker Desktop within their organization.
```
[the remark this answers is not in the document]
Comment (ctalledo) [reply content/desktop/hardened-desktop/_index.md:55]: I personally like the grid pattern very much, it's much easier to read than bullets. In fact I would encourage us to use more grids & figures in the docs, as they are pretty wordy right now :)

File: content/security/_index.md [hunk 3]
```diff
@@ -19,6 +19,10 @@ grid_admins:
   description: Control the images developers can pull from Docker Hub.
   icon: photo_library
   link: /security/for-admins/image-access-management/
+- title: "Air-Gapped Containers"
+  description: Restrict containers from accessing unwanted network resources.
+  icon: "security"
```
[the remark this answers is not in the document]
Comment (ctalledo) [reply content/security/_index.md:24]: Thanks for the suggestion; I like the [vpn_lock](https://fonts.google.com/icons?icon.size=24&icon.color=%23e8eaed&icon.query=vpn_lock) better. 

@djs55 what do you think?

File: content/desktop/hardened-desktop/_index.md [hunk 4]
```diff
@@ -8,19 +8,23 @@ grid:
...
     description: Control the registries developers can access while using Docker Desktop.
     icon: "home_storage"
-    link: "/security/for-admins/registry-access-management/"
+    link: /security/for-admins/registry-access-management/
   - title: "Image Access Management"
     description: Control the images developers can pull from Docker Hub.
     icon: "photo_library"
-    link: "/security/for-admins/image-access-management/"
+    link: /security/for-admins/image-access-management/
+  - title: "Air-Gapped Containers"
```
[the remark this answers is not in the document]
Comment (ctalledo) [reply content/desktop/hardened-desktop/_index.md:24]: Since this is in the grid, why not capitalize the letters as we do in the other grid elements?

File: content/security/_index.md [hunk 5]
```diff
@@ -19,6 +19,10 @@ grid_admins:
   description: Control the images developers can pull from Docker Hub.
   icon: photo_library
   link: /security/for-admins/image-access-management/
+- title: "Air-Gapped Containers"
```
[the remark this answers is not in the document]
Comment (ctalledo) [reply content/security/_index.md:22]: Same as above: since this is in the grid, why not capitalize the letters as we do in the other grid elements?

File: content/desktop/hardened-desktop/_index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (ctalledo) [reply content/desktop/hardened-desktop/_index.md:53]: OK thanks, I've moved the grid to where it was. I think if we make the page more concise (follow-up PR), it would help as right now it's a bit wordy for an intro page.

File: content/desktop/hardened-desktop/_index.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (ctalledo) [reply content/desktop/hardened-desktop/_index.md:24]: Yes it's a feature in it's own right, so I've kept the capitalization in the grid.

File: content/security/_index.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (ctalledo) [reply content/security/_index.md:24]: I've updated the icon to vpn_lock.

**Comment by ctalledo** [comment]
Hi Allie (@aevesdocker), I've resolved all comments, please take another look when you get a chance. Thanks!
